oxidation/crates/parsec_api_crypto/Cargo.toml

```diff
@@ -24,8 +23,19 @@ x25519-dalek = {version="1.2.0", optional=true}
 crypto_box = {version="0.7.0", optional=true, features=["std"]}
 blake2 = {version="0.9.2", optional=true}
 
-[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dependencies]
-getrandom = {version="0.2.3", features=["js"]}
+# Cryptographic randomness is required for generating SecretKey, SigningKey and PrivateKey
+# For SecretKey, we have `crypto_box` -> [...] -> `rand_core~=0.6` -> `getrandom~=0.2`
+# For SingingKey&PrivateKey we have `<dalek stuff>` -> `rand~=0.5` -> `getrandom~=0.1`
+# So we endup with two version of `getrandom` which have each they own way of
+# configuring wasm-unknown-unknown web support (see [features] part).
+getrandom_01 = {package="getrandom", version="0.1.16", optional=true}
+getrandom_02 = {package="getrandom", version="0.2.3", optional=true}
+# On top of that we need to have access to the two version of rand (0.7 and 0.8)
+# to provide the randmoness configuration to the crypto functions.
+# rand 0.7 relies on rand_core~=0.5/getrandom~=0.1
+rand_07 = {package="rand", version="0.7", optional=true}
+# rand 0.8 relies on rand_core~=0.6/getrandom~=0.2
+rand_08 = {package="rand", version="0.8", optional=true}
```

Comment:
So the pattern is pattern is `package_MAJORMINOR`? Maybe `package_MAJOR_MINOR` is more flexible?

Reply:
the name can be changed as needed anytime so I think it's no big deal ;-)